PowerPointLabs/PowerPointLabs/PasteLab/ReplaceWithClipboard.cs

```diff
@@ -74,10 +90,17 @@ static internal class ReplaceWithClipboard
 
                 List<Format> positionFormats = new List<Format> {new PositionXFormat(), new PositionYFormat()};
                 formatsToIgnore.AddRange(positionFormats);
-                
+
                 for (int i = 1; i <= pastingShapes.Count; i++)
                 {
-                    ShapeUtil.ApplyAllPossibleFormats(selectedChildShape, pastingShapes[i], formatsToIgnore);
+                    if (pastingShapes[i].IsPicture())
+                    {
+                        ShapeUtil.ApplyAllPossibleFormats(selectedChildShape, pastingShapes[i], formatsToIgnoreIfPicture);
+                    }
+                    else
+                    {
+                        ShapeUtil.ApplyAllPossibleFormats(selectedChildShape, pastingShapes[i], formatsToIgnore);
+                    }
```

Comment:
This logic seems to have appeared twice, but the duplicated code is short and only for 2 occurrences. Refactoring it into a separate function might make it a little less convenient to read though.